client/client.go

```diff
@@ -16,6 +16,7 @@ type Client struct {
 
 	// Backend interface
 	Backend backend.BackendAPI
+	Append  []string
```

Comment:
There is no need to add `Append` here, on the Client level.
You can transfer the array as method args https://github.com/AirHelp/treasury/blob/6b7a6ce817c6f96f91c2b6f818c69b18c39503f7/cmd/template.go#L70